app/src/main/res/layout/activity_photo_view.xml

```diff
@@ -20,7 +20,7 @@
         android:visibility="visible" />
 
 
-    <ImageView
+    <uk.co.senab.photoview.PhotoView
```

Comment:
Ya I will change it. Someother dependency should've been already present in the app that has additional feature of zooming then.